gsuite/data_user.go

```diff
@@ -0,0 +1,269 @@
+package gsuite
+
+import (
+  "strings"
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+  directory "google.golang.org/api/admin/directory/v1"
+)
+
+func dataUser() *schema.Resource {
+	return &schema.Resource{
+		Read: dataUserRead,
+		Schema: map[string]*schema.Schema{
```

Comment:
too bad there's not a nicer way I know of to tie this schema to `resourceUser()`.  Seems like mostly the same schema just more `Computed` values here.

Reply:
they may also deviate further in the future :/